R/build.r

```diff
@@ -309,6 +309,23 @@
 #'    deploy:
 #'      install_metadata: true
 #'    ```
+#' @section YAML config - footer:
+#' By default, the footer is automatically populated with:
+#' * the names of the authors `authors`, on the left;
+#' * a reference to pkgdown `pkgdown`, on the right.
```

Comment:
I think you should mention that unlike the navbar and sidebar, these are just pasted together into a string.